tests/VersionParserTest.php

```diff
@@ -86,8 +86,12 @@ public function successfulNormalizedVersions()
             'parses arbitrary/3' => array('dev-feature/foo', 'dev-feature/foo'),
             'ignores aliases' => array('dev-master as 1.0.0', '9999999-dev'),
             'semver metadata' => array('dev-master+foo.bar', '9999999-dev'),
-            'semver metadata/2' => array('1.0.0-beta.5+foo', '1.0.0.0-beta5'),
+            'semver metadata/2' => array('1.0.0-beta.5+foo', '1.0.0.0-beta.5'),
             'semver metadata/3' => array('1.0.0+foo', '1.0.0.0'),
+            'semver metadata/4' => array('1.0.0-alpha.3.1+foo', '1.0.0.0-alpha.3.1'),
```

Comment:
That's not what I am talking about, look above you had to change the output of an existing passing test to make it work after this patch, so it did change existing behavior. I'm talking about the leading dot, doesn't matter how many digits there are.